### PR TITLE
Create testdriver_generate_test_report.md

### PR DESCRIPTION
**Pull request by paulmeyer90**
From #7

### CODE REVIEW

File: rfcs/testdriver_generate_test_report.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Summary
+I propose to add TestDriver support for the "generate_test_report" WebDriver command, which will allow generic reports to be consistently generated during web platform tests. This command is specced as part of the Reporting API spec [here](https://w3c.github.io/reporting/#generate-test-report-command).
```
Comment (foolip) [inline rfcs/testdriver_generate_test_report.md]:
```suggestion
Add TestDriver support for the "generate_test_report" WebDriver command, which will allow generic reports to be consistently generated during web platform tests. This command is specced as part of the Reporting API spec [here](https://w3c.github.io/reporting/#generate-test-report-command).
```

File: rfcs/testdriver_generate_test_report.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Motivation
+The real-world causes of reports (like deprecations, interventions, and crashes) can differ between different user agents. In order to test reporting consistently, we need the ability to generate reports universally across the web platform. This command provides that ability.
+
+# Risks
+I am not aware of any risks.
```
Comment (foolip) [inline rfcs/testdriver_generate_test_report.md]:
```suggestion
No known risks.
```

**Comment by jgraham** [comment]
I think this is probably fine. One "risk" that isn't mentioned is that the spec is unstable so the API may undergo semantic changes. I assume that the people working on this feature would keep the testdriver implementation up to date with any spec changes? (this is of course a fairly generic risk with testing things that aren't yet shipping in multiple implementations).

**Comment by foolip** [comment]
@jgraham do you want to suggest an edit to capture that?

**Comment by foolip** [comment]
@jgraham I went to suggest the edits myself, and found the Advantages section hard to do anything with. So I sent https://github.com/web-platform-tests/rfcs/pull/14 and changed the structure of this to conform. Can you review both?

**Comment by foolip** [comment]
@sideshowbarker merged https://github.com/web-platform-tests/rfcs/pull/14, so I'll go ahead and merge this. Will just add the RFC number first.

**Pull request merged**